### PR TITLE
Add manual: Change the Storage Class On an Existing Volume

### PR DESCRIPTION
**Pull request by futuretea**
Signed-off-by: futuretea <Hang.Yu@suse.com>

**Related issues**
https://github.com/harvester/harvester/issues/2838

### CODE REVIEW

File: docs/volume/edit-volume.md [hunk 1]
```diff
@@ -20,4 +20,11 @@ To prevent the expansion from interference by unexpected data R/W, Harvester sup
 
 If you specify a size larger than Longhorn's capacity during the expansion, the status of the volume expansion will be stuck in `Resizing`. You can cancel the failed volume expansion by clicking the `⋮` button and selecting the `Cancel Expand` option.
 
-![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
\ No newline at end of file
+![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
+
+## Change the Storage Class On an Existing Volume
```
Comment (vickyhella) [inline docs/volume/edit-volume.md:25]:
```suggestion
## Change the Storage Class of an Existing Volume
```

File: docs/volume/edit-volume.md [hunk 2]
```diff
@@ -20,4 +20,11 @@ To prevent the expansion from interference by unexpected data R/W, Harvester sup
...
+
+The storage class on an existing image volume cannot be changed, but for other volumes, you can change the storage class by following the steps below
```
Comment (vickyhella) [inline docs/volume/edit-volume.md:27]:
```suggestion
The storage class of an existing image volume cannot be changed. However, you can change the storage class of other volumes by following the steps below:
```

File: docs/volume/edit-volume.md [hunk 3]
```diff
@@ -20,4 +20,11 @@ To prevent the expansion from interference by unexpected data R/W, Harvester sup
...
+
+- [Take a volume snapshot](./volume-snapshots.md#create-volume-snapshots)
+- [Using other storage classes to restore the volume snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot)
```
Comment (vickyhella) [inline docs/volume/edit-volume.md:30]:
```suggestion
1. [Take a volume snapshot](./volume-snapshots.md#create-volume-snapshots).
2. [Use other storage classes to restore the volume snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot).
```

**Review by vickyhella — COMMENTED** [review]
Just some nits

**Review by vickyhella — APPROVED** [review]

File: docs/volume/edit-volume.md [hunk 4]
```diff
@@ -20,4 +20,11 @@ To prevent the expansion from interference by unexpected data R/W, Harvester sup
 
 If you specify a size larger than Longhorn's capacity during the expansion, the status of the volume expansion will be stuck in `Resizing`. You can cancel the failed volume expansion by clicking the `⋮` button and selecting the `Cancel Expand` option.
 
-![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
\ No newline at end of file
+![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
+
+## Change the Storage Class of an Existing Volume
```
Comment (guangbochen) [inline docs/volume/edit-volume.md:25]: s\Storage Class\StorageClass
can we change this word in all places to align with the k8s' official doc? thanks.

File: docs/volume/edit-volume.md [hunk 5]
```diff
@@ -20,4 +20,11 @@ To prevent the expansion from interference by unexpected data R/W, Harvester sup
...
+
+The storage class of an existing image volume cannot be changed. However, you can change the storage class of other volumes by following the steps below:
+
+1. [Take a volume snapshot](./volume-snapshots.md#create-volume-snapshots).
+2. Use another storage class to [restore the volume snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot).
```
Comment (guangbochen) [inline docs/volume/edit-volume.md:30]:
```suggestion
2. Select StorageClass when [restoring the volume using snapshot](./volume-snapshots.md#restore-a-new-volume-using-volume-snapshot).
```

File: docs/volume/edit-volume.md [hunk 6]
```diff
@@ -20,4 +20,11 @@ To prevent the expansion from interference by unexpected data R/W, Harvester sup
 
 If you specify a size larger than Longhorn's capacity during the expansion, the status of the volume expansion will be stuck in `Resizing`. You can cancel the failed volume expansion by clicking the `⋮` button and selecting the `Cancel Expand` option.
 
-![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
\ No newline at end of file
+![cancel-failed-volume-expansion](/img/v1.2/volume/cancel-failed-volume-expansion.png)
+
+## Change the Storage Class of an Existing Volume
+
+The storage class of an existing image volume cannot be changed. However, you can change the storage class of other volumes by following the steps below:
```
Comment (guangbochen) [inline docs/volume/edit-volume.md:27]:
```suggestion
The StorageClass of an existing volume cannot be changed. However, you can change the StorageClass while restoring a new volume from the snapshot by following the steps below:
```

**Review by guangbochen — CHANGES_REQUESTED** [review]
Also need to backport to v1.1.

File: docs/volume/edit-volume.md (hunk 4 above)
Comment (futuretea) [reply docs/volume/edit-volume.md]: Renamed. Screenshots in the docs and the UI also need to be updated.

**Review by guangbochen — APPROVED** [review]
LGTM, thanks.

**Pull request merged**